libs/coin-framework/src/api/types.ts

```diff
@@ -31,6 +31,14 @@ export type Asset = {
   native: bigint;
 };
 
+export type TransactionIntent = {
+  sender: string;
+  recipient: string;
+  amount: bigint;
+  contractAddress?: string;
```

Comment:
`contractAddress` is for the token smartcontract address?
It may be more explicit to call it `tokenAddress` even if for trc10 it is not an address.

Reply:
in the case of `Tron` we do not use it (or in any other coin modules right now) but `Intent` was defined like this in the doc. Should I keep it and just rename it `tokenAddress`